packages/client/components/TeamPrompt/WorkDrawer/GCalIntegrationResults.tsx

```diff
@@ -94,6 +96,12 @@ const GCalIntegrationResults = (props: Props) => {
             <div className='mt-7 w-2/3 text-center'>
               Looks like you don’t have any events to display
             </div>
+            <Link
+              to={`/team/${teamId}/settings/integrations`}
+              className='mt-4 font-semibold text-sky-500 hover:text-sky-400'
+            >
+              Configure Google Calendar
```

Comment:
+1 I'd only change the copy to something like `Not seeing the data? Check the `integration` configuration`

Reply:
Hmm this will be shown both in empty state and in error state, so I'm not sure "Not seeing the data?" is a phrase that belongs here.

@enriquesanchez thoughts on this? Maybe we go with just `Check the <integration> configuration`?

Reply:
Yeah, how does `Review your <integration > configuration` sound?

Reply:
SGTM!